submissions/martthaa/popUp/style.css

```diff
@@ -0,0 +1,224 @@
+  appearance: none;
+}
+
+.box-navigation-buttons{
+  top: 40px;
+  right: 40px;
+  gap: 4em;
+  display: inline-flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: space-around;
+  font-family: "Poppins", sans-serif;
+}
+
+.list-item {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+}
+
+.icon{
+  width: 140px;
+  height: 140px;
+  display: block;
+  margin: 0 auto;
+  border-radius: 50%;
+}
+
+.popup{
+  position: absolute;
+  right: 80px;
+  top: 10px;
+  width: 400px;
+  max-height: 480px;
+  border-radius: 20px;
+  transition: all 0.5s;
+  background-color:  rgba(215, 207, 196, 0.5);
+  box-shadow: 0px 0px 10px 1px #808080;
+  display: flex;
+  flex-wrap: wrap;
+  opacity: 0;
+  visibility: hidden;
+  overflow-y: auto;
+  padding: 20px 10px;
+}
+
+.check_input:checked ~ .popup {
+  opacity: 1;
+  visibility: visible;
+}
+
+.popup_list {
+  display: flex;
+  flex-wrap: wrap;
+  justify-content: space-around;
+  padding: 0.4em;
+  font-size: 18px;
+}
+
+.popup_btn, .link_more {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  left: 0;
+  height: 40px;
+  width: 100%;
+  cursor: pointer;
+  font-family: 'Brush Script MT', cursive;
+  margin-top: 15px;
+  padding: 25px 0;
+  font-size: 40px;
+  text-decoration: none;
+}
+
+.more-item-menu {
+  visibility: hidden;
+  opacity: 0;
+  position: absolute;
+}
+
+.popup_item {
+  width: 9em;
+  height: 9.5em;
+  display: inline-flex;
+  justify-content: center;
+  align-items: center;
+  margin-bottom: 30px;
+}
+
+.popup_link {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  gap: 10px;
+  font-family: 'Brush Script MT', cursive;
+  font-size: 30px;
+}
+
+.popup_more:checked ~ .more-item-menu {
+  opacity: 1;
+  visibility: visible;
+  position: relative;
+  border-radius: 5px;
+  transition: all 0.5s;
+  box-shadow: 0px;
+}
+
+.popup_btn, .link_more{
+    color: #000000 ;
+}
+
+.popup_btn:hover, .link_more:hover{
+    color: #808080;
+}
+
+.popup_more:checked ~ .popup_btn {
+  opacity: 0;
+  visibility: hidden;
+}
+
+.popup_link:hover{
+  color: #808080;
+}
+
+.last-item-menu{
+    padding-top: 35px;
+}
+
+.list-item:hover, 
+.list-item:focus-within   {
+    border: 1px solid #ffffff;
+    box-shadow: 0 0 10px #808080;
+    border-radius: 5px;  
+}
+
+.list-item {
+  border: 1px solid transparent;
+}
+
+.button-icon{
+  width: 40px;
+  height: 40px;
+  cursor: pointer;
+}
+
+.avatar {
+  width: 65px;
+  height: 65px;
+  border-radius: 50%;
+}
+
+@media screen and (max-width:540px) {
+  .popup{
+    right: -80px;
+    top: 100px;
+    padding: 30px;
+  }
+
+  .box-navigation-buttons{
+    flex-direction: row;
+  }
+
+  .popup_item{
+    margin-top: 10px;
+  }
+}
+
+@media screen and (max-width:462px) {
+  .popup{
+    right: -110px;
+    width: 300px;
+    max-height: 480px;
+  }
+
+  .last-item-menu{
+    margin-top: 35px;
+  }
+}
+
+@media screen and (max-width:375px) {
```

Comment:
why do you need so many breakpoints? it is a good practice to use several common breakpoints
https://www.lambdatest.com/blog/how-to-use-css-breakpoints-for-responsive-design/